packages/enzyme-adapter-react-13/src/ReactThirteenAdapter.js

```diff
@@ -125,13 +125,16 @@ class ReactThirteenAdapter extends EnzymeAdapter {
       render(el, context, callback) {
         if (instance === null) {
           const { ref, type, props } = el;
+          const ReactWrapperComponent = createMountWrapper(el, { ...options, adapter });
+          const refProp = ReactWrapperComponent.supportsForwardedRef === true
```

Comment:
i'm not sure if this is sufficient - enzyme itself doesn't care about adapter-utils. the issue is that there needs to be a way for *new enzyme* to opt in to this new behavior, so that by default, it continues to have the old behavior.

Reply:
The old behavior is a bug though not just a feature. I don't think I understand the problem with this approach. Older versions of `enzyme-adapter-utils` will continue to attach the ref to the wrapper component. Only newer versions will correctly forward the ref.

Reply:
`ReactWrapperComponent.supportsForwardedRef` will never not be true though, because `enzyme-adapter-utils` is a regular dependency of the adapter - so as part of releasing this change, the react 13 adapter, for example, will require a version of adapter-utils that has `supportsForwardedRef` set to true.

Reply:
Which is not ideal but if we document that you need to update two dependencies to get this bugfix I don't see an issue. Unless the 13 adapter doesn't work with new adapter-utils versions at which point we just skip the 13 adapter and mark it as wontfix.

Reply:
Sorry again, to clarify:
1. to get the bugfix, someone will need to update to enzyme-next, and adapter-next
1. with enzyme-current and adapter-next, nothing should change
1. with enzyme-next and adapter-current, nothing should change

adapter-next will have always have adapter-utils-next.

Reply:
I don't understand how the current approach breaks these combinations. Could you give me a concrete example that would somehow break with the current approach?

Reply:
Sure. First, `supportsForwardedRef` will never be false, because any version of an adapter that has code to check it, will also force a version of adapter-utils that has `supportsForwardedRef` set to true.

So, enzyme calling `render` with a new adapter will always use the new logic.
-----
As I was writing this comment, I've realized that nothing in this PR changes enzyme itself, only the adapter and utils. Are there no changes needed in enzyme itself for this bugfix? Are there any observable behaviors that would be broken?

Separately, will your test changes fail with the older adapters? If not, can we add some tests that would?